source/tyr/tyr/__init__.py

```diff
@@ -88,4 +88,6 @@ def celery_setup_logging(*args, **kwargs):
 
 tyr_rabbit_mq_handler = RabbitMqHandler(app.config['CELERY_BROKER_URL'], "tyr_event_exchange")
 
+JOB_ID_DELIMITER = "<JOB_ID_DELIMITER>"
```

Comment:
This will be written as part of filenames right ? 
It seems strange to me to use "<>" in filenames. Why not  "_JOB_ID_DELIMITER_" ?